### PR TITLE
Requested changes

### PR DESCRIPTION
**Pull request by michaelwitting**
- changed function names
- added defaults
- removed `return()` if not needed

### CODE REVIEW

File: R/adducts.R [hunk 1]
```diff
@@ -0,0 +1,449 @@
+#' @title Calculate mass-to-charge ratio
+#'
+#' @description
+#'
+#' `mass2mz` calculates the m/z value from a neutral mass and an adduct 
+#'      definition.
+#'
+#' @param x `numeric` neutral mass for which the adduct m/z shall be calculated.
+#'
+#' @param adduct `character` Valid adduct definition.
```
Comment (jorainer) [inline R/adducts.R]: Would be good to say what the *valid*/supported adduct definitions are. Something like "valid adduct definition; supported values are returned by [adductNames()]".

File: R/adducts.R [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+#' @title Calculate neutral mass
+#'
+#' @description
+#'
+#' `mz2mass` calculates the neutral mass from a given m/z value and adduct 
+#'      definition
+#'
+#' @param x `numeric` m/z value for which the neutral mass shall be calculated.
+#'
+#' @param adduct `character` Valid adduct definition.
```
Comment (jorainer) [inline R/adducts.R]: You could use here a `@inheritParams mass2mz` and drop the definition of `@param adduct` - roxygen will then use the definition of this parameter in `mass2mz`.

File: R/adducts.R [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+    
+  }
+}
+
+
+#===============================================================================
+# Private functions
+#===============================================================================
+.adductRules <- function(polarity = c("positive", "negative")) {
+
```
Comment (jorainer) [inline R/adducts.R]: I would add here a `match.arg(polarity)` - then you don't need the final `else` and the `stop` below.

**Comment by jorainer** [comment]
Are you still fixing something or can I merge?

**Comment by michaelwitting** [comment]
Yes, just made a few minor correction on the calculations .

**Pull request merged**